SocialCareCaseViewerApi/V1/Domain/RelatedRelationship.cs

```diff
@@ -0,0 +1,14 @@
+namespace SocialCareCaseViewerApi.V1.Domain
+{
+    public class RelatedRelationship
```

Comment:
I'm sure we could name this better but I can't think of anything! 😅

Reply:
Maybe "Relationship" or "Relation" can be used, anyways we can change it when we remove the code for "RelatedPerson"

Reply:
Hmmm, not sure about those as they feel too generic and possibly confusing against other classes. Let's run with this for now and like you said, we can revisit the name.